src/app.js

```diff
@@ -37,22 +39,32 @@ export default (
       });
       return response.sendStatus(204);
     }
-    response.set({ 'Access-Control-Allow-Origin': '*' });
+    response.set({
```

Comment:
i run prettifier again. nothing changes. The multil-ines looks much prettier for me